pkg/reconciler/revision/resources/queue.go

```diff
@@ -85,7 +85,10 @@ var (
 		ReadOnlyRootFilesystem:   ptr.Bool(true),
 		RunAsNonRoot:             ptr.Bool(true),
 		Capabilities: &corev1.Capabilities{
-			Drop: []corev1.Capability{"all"},
+			Drop: []corev1.Capability{"ALL"},
+		},
+		SeccompProfile: &corev1.SeccompProfile{
```

Comment:
I guess this takes precedence when it comes to podsecurity.SeccompProfile.

Reply:
Yes, it does.